plugins/MultiNull/src/Logging.cpp

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Why aren't you using the PluginLog object API? Fully C++ and dynamic. Basically the USB/FW/dev9 API.

By the way, you mix configuration in the middle of loggin. For me they're unrelated.